lib/screens/upload_screen.dart

```diff
@@ -111,7 +110,13 @@ class _UploadWidgetState extends State<_UploadWidget> {
             child: GestureDetector(
               onTap: () async {
                 final result = await FileUtils.pickFile(provider.nftFormat);
-                widget.onFilePicked(result);
+                if (result == null) {
+                  ScaffoldMessenger.of(context).showSnackBar(const SnackBar(
+                    content: Text("Unsupported format."),
```

Comment:
Centralize the text please.

Reply:
@kjawadDeveloper2 , thank you for pointing this out. But I didn't think the centralized text would give better look. I'd make this change if anyone else agreed. Hey guys, please advise!